src/Kdyby/Doctrine/Dql/InlineParamsBuilder.php

```diff
@@ -1,107 +0,0 @@
-<?php
-
-/**
- * This file is part of the Kdyby (http://www.kdyby.org)
- *
- * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
- *
- * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
- */
-
-namespace Kdyby\Doctrine\Dql;
-
-use Kdyby;
-use Kdyby\Doctrine\Helpers;
-use Nette;
-
-
-
-/**
- * @author Filip Procházka <filip@prochazka.su>
- */
-class InlineParamsBuilder extends Kdyby\Doctrine\QueryBuilder
```

Comment:
@fprochazka You wanted to remove whole Dql namespace. Where should I move it then?